server/src/__tests__/donatedItem.test.ts

```diff
@@ -0,0 +1,72 @@
+import request from 'supertest';
+import express from 'express';
+import donatedItemRoutes from '../routes/donatedItemRoutes';
+import mockPrismaClient from '../__mocks__/mockPrismaClient';
+import { newProgram } from '../__mocks__/mockProgram';
+import { newDonor } from '../__mocks__/mockDonor';
+import { newItem } from '../__mocks__/mockNewItem';
+import { updateData } from '../__mocks__/mockUpdateData';
+
+const app = express();
+app.use(express.json());
+app.use('/donatedItem', donatedItemRoutes);
+
+describe('DonatedItem API Tests', () => {
+    beforeEach(() => {
+        jest.clearAllMocks();
+        // Setup mock responses
+        mockPrismaClient.program.findUnique.mockResolvedValue(newProgram);
+        mockPrismaClient.donor.findUnique.mockResolvedValue(newDonor);
+        mockPrismaClient.donatedItem.create.mockResolvedValue({ id: 1, ...newItem });
+        mockPrismaClient.donatedItem.update.mockResolvedValue({ id: 1, ...updateData });
+    });
```

Comment:
Nice! Keeping the required mock resolved values for prisma at one place seems more clear